### PR TITLE
FUSETOOLS-853/854 - enable CircleCI builds

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    working_directory: ~/vscode-atlasmap
+    docker:
+      - image: circleci/node:lts-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: test java
+          command: java --version
```
Comment (apupier) [inline .circleci/config.yml:23]: I do not think that this java test version is useful.
i added it on the other circelci job to try to debug for initial state.

**Review by apupier — CHANGES_REQUESTED** [review]
there will be the sonar configuration to do but I think this can be done in a separated PR

File: .circleci/config.yml (hunk 1 above)
Comment (lhein) [reply .circleci/config.yml:23]: agreed. will remove it.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      - run:
+          name: test UI
+          command: npm run ui-test
+      - sonarcloud/scan 
+      - save_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/.npm
+orbs:
+  sonarcloud: sonarsource/sonarcloud@1.0.0
```
Comment (apupier) [inline .circleci/config.yml:49]: the latest is 1.0.2 https://circleci.com/developer/orbs/orb/sonarsource/sonarcloud

Comment (lhein) [reply .circleci/config.yml]: ok

**Comment by lhein** [comment]
Travis can be ignored because it will be disabled once CircleCI is working.

**Review by apupier — APPROVED** [review]

**Review by bfitzpat — APPROVED** [review]

**Pull request merged**